cocos/core/animation/cross-fade.ts

```diff
@@ -71,6 +71,14 @@ export class CrossFade extends Playable {
             }
             this._fadings.splice(deadFadingBegin);
         }
+
+        if (this._managedStates.length !== 0) {
+            this._managedStates.forEach((state) => {
```

Comment:
This is too heavy in an update function, please modify to use simple for loop, so that there won't be any function creation or invocation

Reply:
Updated.